ggshield/sca/output/text_handler.py

```diff
@@ -71,11 +108,31 @@ def process_sca_file_result(
         return result_buf.getvalue()
 
 
+def sca_file_info(
```

Comment:
You can just use `file_info` with an optional `incident_status` argument equal to `"detected"` here I think.

Reply:
Added, with an Enum in constant. I'm not fully conviced by my proposal. WDYT ?

Reply:
I think it's good, what is bothering you in this?

Reply:
I think it's modifying some core element with technical specficities maybe. But let's do it this way !